custom/enikshay/user_setup.py

```diff
@@ -318,6 +319,31 @@ def init_form(self, post_dict=None):
                 fs[i] = _make_field_visible_to(field, fields_to_loc_types[field])
         return form
 
+    def _make_field(self, field):
+        if field.slug == 'language_code':
+            return forms.ChoiceField(
+                label=field.label,
+                required=True,
+                choices=[
+                    ('', _('Select one')),
+                    ("en", "English"),
+                    ("hin", "Hindi"),
+                    ("mar", "Marathi"),
+                    ("bho", "Bhojpuri"),
+                    ('guj', "Gujarati"),
+                ],
+            )
+        if field.slug == 'contact_phone_number':
+            regexp = "^91[1-9][0-9]{9}$"
```

Comment:
I checked in the app, and that validation condition did require the leading number to be 1-9